action/report/htmlreporter.go

```diff
@@ -53,57 +54,6 @@ func (ht *HTMLReporter) Gen(out io.Writer) error {
 	return nil
 }
 
-// HTMLHistCharts returns histogram charts section in HTML report
```

Comment:
I'm confused. Are these changes related to the lint report? If not, what is being attempted here?

Reply:
Which changes? HTMLHistCharts was an unused func which is now gone.

Reply:
unused. ah. ok. didn't realize; thought perhaps there were some report changes as well.